core/src/main/java/org/geysermc/geyser/registry/type/ItemMappings.java

```diff
@@ -134,6 +135,8 @@ public ItemMapping getMapping(ItemData data) {
             return ItemMapping.AIR;
         } else if (definition.getRuntimeId() == lodestoneCompass.getBedrockDefinition().getRuntimeId()) {
             return lodestoneCompass;
+        } else if (lightBlocks.containsKey(definition.getRuntimeId())) {
```

Comment:
This can be moved into its own `if` statement. `lightBlock = lightBlocks.get(definition.getRuntimeId()); if (lightBlock != null) return lightBlock;` (but with proper format)